scripts/test/ProcessManagement/TransitionAction/AppointmentUpdate.t

```diff
@@ -0,0 +1,131 @@
+# --
+# Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# --
+# This software comes with ABSOLUTELY NO WARRANTY. For details, see
+# the enclosed file COPYING for license information (AGPL). If you
+# did not receive this file, see http://www.gnu.org/licenses/agpl.txt.
+# --
+
+use strict;
+use warnings;
+use utf8;
+
+use vars (qw($Self));
+
+use Kernel::System::VariableCheck qw(:all);
+
+$Kernel::OM->ObjectParamAdd(
+    'Kernel::System::UnitTest::Helper' => {
+        RestoreDatabase => 1,
+    },
+);
+
+my $AppointmentObject  = $Kernel::OM->Get('Kernel::System::Calendar::Appointment');
+my $CalendarObject     = $Kernel::OM->Get('Kernel::System::Calendar');
+my $ConfigObject       = $Kernel::OM->Get('Kernel::Config');
```

Comment:
Is this needed?
Please also note the same comments as in the UnitTest AppointmentRemove.t here